docs/sources/getting-started/glossary.md

```diff
@@ -69,6 +75,12 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
       A series of measurements, ordered by time. Time series are stored in data sources and returned as the result of a query.
     </td>
   </tr>
+  <tr>
+    <td style="vertical-align: top">Trace</td>
+    <td>
+      An observed execution path of a request through a distributed system.
```

Comment:
This is an extremely concise and good definition for a very complex idea.  Does it make sense to link to the OpenTracing "what is tracing" page for more details:

https://opentracing.io/docs/overview/what-is-tracing/

Reply:
Makes sense 👍